apps/costcontrol/js/costcontrol.js

```diff
@@ -643,7 +647,15 @@ var CostControl = (function() {
       if (apps && apps.length > 0) {
         requests = [];
         apps.forEach(function(manifestURL) {
-          requests.push(requestSamples({ appManifestURL: manifestURL }));
+          if (manifestURL === Common.SYSTEM_MANIFEST) {
```

Comment:
Furtheremore, add a comment to explain what's happening: `Ignores the system app because...`